templates/plugin/python/module.conf.erb

```diff
@@ -11,7 +11,11 @@
     <%= key %> <%= k %> <% if !!v == v %><%= v %><% else %>"<%= Shellwords.split(v).join('" "') %>"<% end %>
   <%- end -%>
   <%- else -%>
-    <%= key %> <% if !!value == value %><%= value %><% else %>"<%= Shellwords.split(value).join('" "') %>"<% end %>
+  <%- if not !!value == value and value.include?('"') -%>
```

Comment:
@igalic had suggested shellwords as a way to quote things without knowing if they had quotes around them. For some reason it's not working under jruby in particular.